Services/Dashboard/Administration/classes/class.ilObjDashboardSettingsGUI.php

```diff
@@ -114,11 +123,86 @@ public function editSettings(): void
+        $this->tpl->setContent($ui->renderer()->renderAsync($form));
+    }
+
+    public function getViewForm(string $mode): StandardForm
+    {
+        switch ($mode) {
+            case self::VIEW_MODE_PRESENTATION:
+            case self::VIEW_MODE_SORTING:
+                return $this->ui_factory->input()->container()->form()->standard(
+                    $this->ctrl->getFormAction($this, 'save' . $mode),
+                    array_map(
+                        fn (int $view): Section =>
+                        $this->getViewByMode(
+                            $mode,
+                            $view
+                        ),
+                        $this->viewSettings->getPresentationViews()
+                    )
+                );
+            case self::VIEW_MODE_SETTINGS:
+            default:
+                return $this->getSettingsForm();
+        }
+    }
+
+    public function getViewSectionSorting(int $view, string $title): Section
+    {
+        $this->tpl->addJavaScript("Services/Dashboard/Administration/js/SortationUserInputHandler.js");
+        $lng = $this->lng;
+        $availabe_sort_options = $this->viewSettings->getAvailableSortOptionsByView($view);
+        $options = array_reduce(
+            $availabe_sort_options,
+            static function (array $options, string $option) use ($lng): array {
+                $options[$option] = $lng->txt("dash_sort_by_" . $option);
+                return $options;
+            },
+            []
+        );
+
+        $available_sorting = $this->ui_factory
+            ->input()
+            ->field()
+            ->multiSelect($this->lng->txt("dash_avail_sortation"), $options)
+            ->withValue(
+                $this->viewSettings->getActiveSortingsByView($view)
+            )
+            ->withAdditionalOnLoadCode(
+                static fn (string $id) =>
+                    "document.getElementById('$id').setAttribute('data-checkbox', 'activeSorting$view');
+                    document.addEventListener('DOMContentLoaded', function () {
+                        handleUserInputForSortationsByView($view);
+                    });"
+            );
+        $default_sorting = $this->ui_factory
+            ->input()
+            ->field()
+            ->select($this->lng->txt("dash_default_sortation"), $options)
+            ->withValue($this->viewSettings->getDefaultSortingByView($view))
+            ->withRequired(true)
+            ->withAdditionalOnLoadCode(
+                static fn (string $id) =>
+                    "document.getElementById('$id').setAttribute('data-select', 'sorting$view');"
+            );
+        return $this->ui_factory->input()->field()->section(
+            $this->maybeDisable(["avail_sorting" => $available_sorting, "default_sorting" => $default_sorting]),
+            $title
+        );
+    }
+
+    public function getSettingsForm(): StandardForm
     {
         $ui = $this->ui;
         $f = $ui->factory();
```

Comment:
Im fine with aliases but but they still should keep some semantical context.
Single Char variables are mostly reserved for loop indexes and math operations ($f would be a great variable to store a linear function in or some Fahrenheit temperature).

Maybe as a general rule you could take that ouy always look on the right side of an alias to improve its length, than on the left.
In this special case for example you could alias not only the factory, but for the most cases the input->field ,too:
```
$field = $this->ui->factory()->input()->field()
```

Further the usage of aliases should provide a significant reduction of code.
The alias `$ui` has such less occurences that it actualy inflates the code :smile: 
I personally use "normal" aliases only for > 3 Occurences.